src/applications/hca/reducer.js

```diff
@@ -35,12 +36,21 @@ export function hcaEnrollmentStatus(state = initialState, action) {
 
     case FETCH_ENROLLMENT_STATUS_SUCCEEDED: {
       const {
-        parsedStatus: enrollmentStatus,
+        parsedStatus,
         applicationDate,
         effectiveDate: enrollmentStatusEffectiveDate,
         enrollmentDate,
         preferredFacility,
       } = action.data;
+
+      let enrollmentStatus = parsedStatus;
+      if (enrollmentStatus === HCA_ENROLLMENT_STATUSES.activeDuty) {
```

Comment:
This is really the most significant change. Since all of the existing message-generating code simply looks at the `enrollmentStatus` in the redux store I'm adding two new valid values and will determine which value to use in the reducer.

Reply:
great